lib/deploy_complexity/checklists.rb

```diff
@@ -167,6 +167,23 @@ def relevant_for(files)
     end
   end
 
+  class DockerfileChecklist < Checklist
+    def human_name
+      "Dockerfile"
+    end
+
+    def checklist
+      '
+- [ ] If you added a dependency to the Dockerfile, also add it to Chef so Jenkins master gets the update.
```

Comment:
Perhaps we can make this a little more explicit? Though this is likely too wordy, maybe linking to a page in the wiki may be better and we can give clearer steps?

"If you added a dependency to the Dockerfile for a script that will be called during both CI builds **AND** Deploy builds then you should also add that dependency to the chef recipe for [jenkins_common](https://github.com/NoRedInk/NoRedInk-chef/blob/master/site-cookbooks/noredink/recipes/jenkins_common.rb)."

Reply:
I was going to add the Chef recipe but I'm not sure if we should make that public in this public repo. @dgtized had some concerns there; what do you think Charles?

Reply:
I don't think there is a security issue with linking to the private chef repo, a little hesitant if it's full path, but it's already in this issue which is public so... It does suggest that maybe we should inject the more specific checklist items into the monolith and hook them from here.

Reply:
agreed. I have been meaning to do that, but haven't got around to it yet 😕